Python/flowgraph.c

```diff
@@ -1345,63 +1345,44 @@ add_const(PyObject *newconst, PyObject *consts, PyObject *const_cache)
 
 /*
    Walk basic block backwards starting from "start" trying to collect "size" number of
-   subsequent constants from instructions loading constants into new tuple ignoring NOP's in between.
+   subsequent instructions that load constants into instruction array "instrs" ignoring NOP's in between.
+   Caller must make sure that length of "instrs" is sufficient to fit in at least "size" instructions.
```

Comment:
Too long sentence, it's hard to read. How about:

Traverse the instructions of the basic block backwards from index "start", skipping over NOPs.
Try to collect "size" number of consecutive instructions that load constants into the array "instrs".
Caller must make sure that length of "instrs" is sufficient to fit in at least "size" instructions.